open-company/open-company-web
# Reorder sections

card: https://trello.com/c/kf0gTEPF

To test:
- try to reorder the sections in all the categories
- try to remove some sections in all the categories
- try to move a section in another category: it shouldn't be possible
- run tests or check CI
- merge
